fs/dafsclient/rmtclient.cpp

```diff
@@ -879,35 +896,41 @@ void CRemoteBase::sendRemoteCommand(MemoryBuffer & src, MemoryBuffer & reply, bo
 
         if (!socket)
         {
-            bool doConnect = true;
-            if (connectMethod == SSLFirst || connectMethod == UnsecureFirst)
+            if (storageSecret)
+                connectSocket(tep, 0, INFINITE, true);
```

Comment:
Should timeout be INFINITE or 0 (to use configured default) ?

Reply:
timeout is 0.
retries is INFINITE.

```
    void connectSocket(SocketEndpoint &ep, unsigned connectTimeoutMs=0, unsigned connectRetries=INFINITE, bool secure=false);
```

Same as defaults, just explicit now.

Reply:
ok, sorry I read the args wrong.